guide/daily-spending-wallet/requesting.md

```diff
@@ -34,440 +34,400 @@ Illustration source
-   layout = "float-left-desktop -background -shadow"
-   caption = "Invoice expiries should be adjusted for the users use case."
-   alt-text = "Screen showing single payment request settings."
+   image = "/assets/images/guide/daily-spending-wallet/requesting/Unified.png"
+   retina = "/assets/images/guide/daily-spending-wallet/requesting/Unified@2x.png"
+   modalImage = "/assets/images/guide/daily-spending-wallet/requesting/Unified@2x.png"
+   layout = "shadow"
+   caption = "Unified payment requests combine lightning and on-chain requests, so users don’t have to worry about which to use."
+   alt-text = "Screen showing a unified payment request in QR format."
    width = 250
    height = 541
+   modalWidth = 250
+   modalHeight = 541
 %}
 
-When denominating invoices in fiat, custom expiries should be used to prevent exposure to price volatility.
-
-If requesting a specific amount denominated in fiat, use a shorter expiry and have the invoice refresh each time it expiries. For a wallet designed for in-person payments, a 30 - 60 second expiry works well.
-
-If there is no amount defined on the invoice, the invoice is denominated in bitcoin, or the invoice needs to be shared in a message, then use a longer expiry, like 24 hours.
-
-While it is good to allow users to define their own custom expiries, remember that only advanced users will do this. Choose a sensible default that makes sense for your user-base and how the wallet is intended to be used.
-
-For some background on situations the user might encounter with different invoice expiries, see this [blog post](https://d.elor.me/2022/01/lightning-invoice-expiration-ux-considerations/) from designer Stephen DeLorme.
-
-Using [BOLT12](https://bolt12.org/) offers, invoice expirations will no longer be a point of user friction. Offers are not yet an accepted standard, or widely supported by wallets, so interoperability will be an issue if adopting it.
-
-</div>
-
-### Address types
-
-For on-chain, there are several address types that can be used for single-use payment requests.
-
-<div class="center" markdown="1">
-
 {% include picture.html
-   image = "/assets/images/guide/daily-spending-wallet/requesting/single-use-payment-request-settings-address.png"
-   retina = "/assets/images/guide/daily-spending-wallet/requesting/single-use-payment-request-settings-address@2x.png"
-   layout = "float-right-desktop -background -shadow"
-   caption = "Supporting every address type benefits users and makes your wallet more interoperable."
-   alt-text = "Screen showing different address types."
+   image = "/assets/images/guide/daily-spending-wallet/requesting/Invoice.png"
+   retina = "/assets/images/guide/daily-spending-wallet/requesting/Invoice@2x.png"
+   modalImage = "/assets/images/guide/daily-spending-wallet/requesting/Invoice@2x.png"
+   layout = "shadow"
+   caption = "Let users share the lightning invoice independently."
```

Comment:
Small (?) next to "Share lightning invoice" might be good for users to quickly pull up help on what this means.